.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -32,7 +32,7 @@ body:
     id: helix-log
     attributes:
       label: Helix log
-      description: See `hx -h` for log file path
+      description: See `hx -h` for log file path. If you can reproduce the issue run `RUST_BACKTRACE=1 hx -vvvv` to generate a more detailed log file.
```

Comment:
vvvv is a bit extreme since you'll get logs for tokio internals. Maybe v or vv?

Reply:
I changed it to `vv`. I originally used `v` but @dead10ck suggested the more verbose version. I agree that including tracing output is probably too much so `vv` seems more appropriate.